### PR TITLE
Document additional outgoing RISC event types:

### PR DESCRIPTION
**Pull request by zachmargolis**
- Identifier Changed
- Recovery Activated
- Recovery Information Changed

### CODE REVIEW

File: _data/risc_outgoing.yml [hunk 1]
```diff
@@ -58,3 +60,55 @@
...
+    "https://schemas.openid.net/secevent/risc/event-type/recovery-activated": {
+      "subject": {
+        "subject_type": "iss-sub",
+        "iss": "https://idp.int.identitysandbox.gov",
+        "sub": "<$SUB>"
+      }
+    }
+  }
+
+- friendly_name: Recovery Information Changed
```
Comment (mitchellhenke) [inline _data/risc_outgoing.yml:97]: thanks for doing this 😬

Comment (zachmargolis) [reply _data/risc_outgoing.yml:97]: Ofc! Wanted to update these now that we can generate them more easily

File: _data/risc_outgoing.yml [hunk 2]
```diff
@@ -58,3 +60,55 @@
...
+  event_type: https://schemas.openid.net/secevent/risc/event-type/recovery-information-changed
+  spec_url: "https://openid.net/specs/openid-risc-event-types-1_0-ID1.html#rfc.section.2.9"
+  description: >
+    Login.gov pushes this event when a user changes any of their second factor, for example,
+    adding or removing an email, phone address, TOTP authentication app, backup codes,
```
Comment (mitchellhenke) [inline _data/risc_outgoing.yml:102]: I didn't include events for adding or removing an email since it's not a second factor, should we?

Comment (zachmargolis) [reply _data/risc_outgoing.yml:102]: [The spec](https://openid.net/specs/openid-risc-event-types-1_0-ID1.html#rfc.section.2.9) makes it sound like yes, (emphasis mine):

> Recovery Information Changed signals that the account identified by the subject has changed some of its recovery information. **For example a recovery email address was added or removed.**

**Review by mitchellhenke — APPROVED** [review]

File: _data/risc_outgoing.yml [hunk 3]
```diff
@@ -58,3 +60,55 @@
     }
   }
 
+- friendly_name: Identifier Changed
+  direction: outgoing
+  event_type: https://schemas.openid.net/secevent/risc/event-type/identifier-changed
```
Comment (stevegsa) [inline _data/risc_outgoing.yml:65]: thank you!

Comment (stevegsa) [reply _data/risc_outgoing.yml:65]: Now all you need is `reproof-completed` :-)

File: _data/risc_outgoing.yml [hunk 4]
```diff
@@ -58,3 +60,55 @@
...
+  }
+
+- friendly_name: Recovery Information Changed
+  event_type: https://schemas.openid.net/secevent/risc/event-type/recovery-information-changed
+  spec_url: "https://openid.net/specs/openid-risc-event-types-1_0-ID1.html#rfc.section.2.9"
+  description: >
+    Login.gov pushes this event when a user changes any of their second factor, for example,
+    adding or removing an email, phone address, TOTP authentication app, backup codes,
+    or PIV/CAC.
+    payload_schema:
```
Comment (aduth) [inline _data/risc_outgoing.yml:104]: Should this have been...

```suggestion
  payload_schema:
```

Currently, it's included in the description text.

Comment (zachmargolis) [reply _data/risc_outgoing.yml:104]: whoooops, great catch, thank you, PR to fix https://github.com/18F/identity-dev-docs/pull/181

**Pull request merged**